test/functional/cmdLineTests/ValueBasedClassTest/playlist.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2021, 2021 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../TKG/playlist.xsd">
+	<test>
+		<testCaseName>cmdLineTester_ValueBasedClassTest</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) \
+		-DJARPATH=$(Q)$(TEST_RESROOT)$(D)ValueBasedClassTest.jar$(Q) \
+		-DJAVA_EXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS) $(SQ) \
+		-jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)jep390.xml$(Q) \
+		-nonZeroExitWhenError; \
+		$(TEST_STATUS)</command>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>functional</group>
+		</groups>
+		<subsets>
+		<!-- run for Java 16 and up. Remove this test once value type is enabled -->
```

Comment:
Currently this test is only enabled on Java 16+. Valhalla build won't run on this test, which is expected. There are some talks regarding enabling full testing on Valhalla builds, not sure if there is a way to disable this test on Valhalla builds.

Reply:
I think we will need runtime checks if ValueTypes are enabled. I received some clarification that this feature may be kept for a few releases after Valluetype are released, so we will need to be able to deal with both.

Reply:
It doesn't have to be done in this PR but we need to added a `J9_IS_VALUETYPE` check before printing the error message

Reply:
> this feature may be kept for a few releases after Valluetype are released, so we will need to be able to deal with both.

Currently this feature throws `VirtualMachineError` on monitor enter. When value type is enabled, it throws IMSE. How are we able to support both ?

Reply:
Currently the code does something like:

```
#if JAVA_SPEC_VERSION >= 16
valueTypeIllegalMonitorState:
      ...
#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
	setValueTypeException()
#else /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
	Assert_VM_true(J9_ARE_ALL_BITS_SET(_vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_VALUE_BASED_EXCEPTION));
	setValueBasedException()
#endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
       ...
#endif /* JAVA_SPEC_VERSION >= 16 */
```

we need to do:

```
#if JAVA_SPEC_VERSION >= 16
valueTypeIllegalMonitorState:
      ...

#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
        if (J9CLASS_IS_VT(clazz)) {
	     setValueTypeException()
        } else {
#endif /* defined(J9VM_OPT_VALHALLA_VALUE_TYPES) */
        {
	       Assert_VM_true(J9_ARE_ALL_BITS_SET(_vm->extendedRuntimeFlags2, J9_EXTENDED_RUNTIME2_VALUE_BASED_EXCEPTION));
	      setValueBasedException()
       }
       ...
#endif /* JAVA_SPEC_VERSION >= 16 */
```

Reply:
So this means when VT is fully enabled, there could be classes inside JDK that are not VT, but annotated with `@jdk.internal.ValueBased` (so that the else block above is reachable) ? 

If we want to make that change, I will do that in another PR.

Reply:
> So this means when VT is fully enabled, there could be classes inside JDK that are not VT, but annotated with @jdk.internal.ValueBased (so that the else block above is reachable) ?

Yes, this was the message I received from the expert group. This may go on for a few releases.

> If we want to make that change, I will do that in another PR.

okay sounds good

Reply:
> Yes, this was the message I received from the expert group. This may go on for a few releases.

OK, I see. 

This also means this test may need to be updated in the future along the releases. Because of the new `J9CLASS_IS_VT(clazz)` check to be added, the expected `VirtualMachineError` will be thrown only on value-based class that is not VT.

Reply:
> This also means this test may need to be updated in the future along the releases. 

The way it is currently written yes, that is because j.l.Integer will be converted to a VT. However if you create a custom class with the annotation then I believe you wont need to update the test

Reply:
Yes, that will work.

Reply:
Updated the test classes.